code/renderers/svelte/src/docs/extractArgTypes.ts

```diff
@@ -102,25 +102,26 @@ const parseTypeToControl = (type: JSDocType | undefined): any => {
   if (type.kind === 'type') {
     switch (type.type) {
       case 'string':
-        return { type: 'text' };
-
+        return { control: { type: 'text' } };
       case 'enum':
-        return { type: 'radio' };
+        return { control: { type: 'radio' } };
       case 'any':
-        return { type: 'object' };
+        return { control: { type: 'object' } };
       default:
-        return { type: type.type };
+        return { control: { type: type.type } };
     }
   } else if (type.kind === 'union') {
     // @ts-expect-error TODO: fix, this seems like a broke in package update
     if (Array.isArray(type.type) && !type.type.find((t) => t.type !== 'string')) {
       return {
-        type: 'radio',
+        control: { type: 'select' },
```

Comment:
I think this should stay a `'radio'` type, as that would also be consistent with React. In the ideal world you could say this should be a select if there are 5 or more options, but I'd even argue that this change here would be a breaking change.